config/locales/en.yml

```diff
@@ -31,3 +31,9 @@
 
 en:
   hello: "Hello world"
+  csv:
```

Comment:
These locales should be in a file called `activemodel.en.yml`. See: https://github.com/nimblehq/internal-certification-ruby/blob/main/huey-toby/config/locales/activemodel.en.yml

Rails can automatically load translations based on file names. See: https://guides.rubyonrails.org/i18n.html#lazy-lookup

Reply:
Fixed in 366f0e361c9209f473a876ffbab0e837add6eb33